Formula/gerbil-scheme.rb

```diff
@@ -0,0 +1,42 @@
+class GerbilScheme < Formula
+  desc "Opinionated dialect of Scheme designed for Systems Programming"
+  homepage "https://cons.io"
+  url "https://github.com/vyzo/gerbil/archive/v0.14.tar.gz"
+  sha256 "8525877960ba7a6122d6dfd2bae34ed1437d5231f9e31de97bbe6106c4dfa03a"
+
+  depends_on "gambit-scheme"
+  depends_on "leveldb"
+  depends_on "libyaml"
+  depends_on "lmdb"
+  depends_on "openssl"
+
+  def install
+    cd "src" do
+      ENV.append_path "PATH", "#{Formula["gambit-scheme"].opt_prefix}/current/bin"
+
+      inreplace "std/build-features.ss" do |s|
+        s.gsub! "(enable leveldb #f)", "(enable leveldb #t)"
+        s.gsub! "(enable libxml #f)", "(enable libxml #t)"
+        s.gsub! "(enable libyaml #f)", "(enable libyaml #t)"
+        s.gsub! "(enable lmdb #f)", "(enable lmdb #t)"
+      end
+
+      ENV.prepend "CPPFLAGS", "-I#{Formula["leveldb"].opt_include}"
```

Comment:
I'm surprised these modifications of `CPPFLAGS` and `LDFLAGS` are necessary. brew will add those to the compiler switches transparently because of the `depends_on` above.

Reply:
I'll test without them.
Would like to see the bot do them as well. since was getting missing headers

Reply:
Will test, as some of these were added for errors seen building.

Reply:
Removed and seems to work on build bot. But not when tested locally.